Description: Make boost::system optional, header-only since 1.69, removed in 1.89.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://github.com/mapnik/mapnik/pull/4550

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -170,7 +170,7 @@ if(CMAKE_CXX_STANDARD VERSION_LESS 17)
     set(USE_BOOST_FILESYSTEM ON CACHE BOOL "Use boost::filesystem" FORCE)
 endif()
 if(USE_BOOST_FILESYSTEM)
-    mapnik_find_package(Boost ${BOOST_MIN_VERSION} REQUIRED COMPONENTS filesystem system)
+    mapnik_find_package(Boost ${BOOST_MIN_VERSION} REQUIRED COMPONENTS filesystem OPTIONAL_COMPONENTS system)
 endif()
 
 list(APPEND MAPNIK_COMPILE_DEFS BOOST_REGEX_HAS_ICU)
